sdks/python/apache_beam/runners/interactive/cache_manager.py

```diff
@@ -69,8 +69,8 @@ def read(self, *labels):
       *labels: List of labels for PCollection instance.
 
     Returns:
-      Tuple[List[Any], int]: A tuple containing a list of items in the
-        PCollection and the version number.
+      A tuple containing an iterator for the items in the PCollection and the
```

Comment:
Please leave the types (it makes it easier to follow).

Reply:
I was getting some pydoc errors and removed it. I put them back in as type hints.